cmds/tui.c

```diff
@@ -142,6 +142,7 @@ static const char *help[] = {
 	"g             Show call graph for this function",
 	"R             Show uftrace report",
 	"r             Show uftrace report for this function",
+	"s             Show uftrace report sorted by KEYs",
```

Comment:
Could you change this like below as @namhyung mentioned?
```suggestion
	"s             Sort by the next column in report",
```